tests/test_integration.py

```
@@ -411,41 +409,35 @@ def test_explicit_overwrite_behavior(tmp_path: Path) -> None:
-    # Note: We're using += operator so people get added, not replaced
+
+    # En mode écrasement, la nouvelle description doit remplacer l'ancienne
+    # Note: Nous utilisons l'opérateur = donc les personnes sont ajoutées et accumulent
     final_keywords = final_metadata.get("Keywords", [])
     if isinstance(final_keywords, str):
         final_keywords = [final_keywords]
-    
-    # Both original and new person should be present (because += adds)
+
+    # Les deux personnes, originale et nouvelle, devraient être présentes (car = accumule pour les listes)
     assert "Original Person" in final_keywords
```

Ruff (0.12.2)

Use of `assert` detected

(S101)